libs/components/forms/src/lib/modules/toggle-switch/toggle-switch.component.html

```diff
@@ -34,15 +36,19 @@
   </button>
   <span
     ><label
-      *ngIf="hasLabelComponent"
+      *ngIf="hasLabelComponent || labelText"
       [for]="toggle.id"
       [attr.id]="labelId"
       class="sky-toggle-switch-label"
-      ><ng-container *ngTemplateOutlet="labelContent"></ng-container></label
+      ><ng-container *ngIf="labelText; else labelContent">
+        <span *ngIf="!labelHidden">{{ labelText }}</span>
```

Comment:
I'd use another `ng-container` instead of `span` so we don't add unnecessary HTML elements to the component.